packages/core/admin/server/config/admin-actions.js

```diff
@@ -180,5 +180,21 @@ module.exports = {
       section: 'settings',
       category: 'project',
     },
+    {
+      uid: 'transfer.push',
```

Comment:
We should probably move that to a bootstrap file in `@strapi/data-transfer` to let the package owns what it exports & register.

We can do smth similar do what we're doing in other packages (eg: [`upload`](https://github.com/strapi/strapi/blob/main/packages/core/upload/server/bootstrap.js#L44))

And then call the data-transfer bootstrap in the admin bootstrap.

What do you think?